storybook/stories/playground/zoom-out/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { registerCoreBlocks } from '@wordpress/block-library';
+import { useDispatch } from '@wordpress/data';
+import {
+	BlockEditorProvider,
+	BlockCanvas,
+	store as blockEditorStore,
+	BlockList,
+} from '@wordpress/block-editor';
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+import { parse } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import editorStyles from '../editor-styles';
+// eslint-disable-next-line @wordpress/dependency-group
+import contentCss from '!!raw-loader!../../../../packages/block-editor/build-style/content.css';
+import { pattern } from './pattern';
+
+// Temporary hack to access private APIs before stabilizing zoom level.
+const { unlock } = __dangerousOptInToUnstableAPIsOnlyForCoreModules(
+	'I acknowledge private features are not for use in themes or plugins and doing so will break in the next version of WordPress.',
+	'@wordpress/edit-site'
+);
+
+function EnableZoomOut() {
+	const { setZoomLevel } = unlock( useDispatch( blockEditorStore ) );
+
+	useEffect( () => {
+		setZoomLevel( 50 );
+	}, [ setZoomLevel ] );
+
+	return null;
+}
+
+export default function EditorZoomOut() {
+	const [ blocks, updateBlocks ] = useState( [] );
+
+	useEffect( () => {
+		registerCoreBlocks();
+		updateBlocks( parse( pattern ) );
+	}, [] );
+
+	return (
+		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
+		<div
+			className="editor-zoom-out"
+			onKeyDown={ ( event ) => event.stopPropagation() }
+			style={ { background: '#ddd', border: '1px solid gray' } }
```

Comment:
The background style shouldn't be necessary, at the moment when the scaling is applied, the "iframe" content is scaled but there's still a "white" background around the iframe content, I believe this style should be bundled within the block-editor package and the iframe should have the same gray background applied within the iframe.